js/project.js

```diff
@@ -0,0 +1,37 @@
+//opening > 
+//on click nav apply .current class to this 
+//On generate # of paragrahs is returned, selected ipsum revealed with slide
+//logic = take value and return that number of p from clicked ipsum
+//click or tab into form, field needs to clear, ipsum disappears with slide
+//define # value
+//function to return text
+
+$("nav a").click(function(){
+    $("nav. current").removeClass(".current");
+    $(this).addClass(".current");
+  });
+
+var 
+$(#Submit).click(function(){
+  var = numParagraphs ("paragraphs" +.value);
+}
```

Comment:
This is a floating closing `}`. Or if it should be here it is missing the closing `);` that should come right after it.